JacobianMatrix[] jacobians
string[] names  # Names of the links/end-effectors corresponding to each Jacobian
